lib/Util/ExtAPI.cpp

```diff
@@ -69,17 +72,25 @@ static void fillExtAPIStringMap() {
 void ExtAPI::init() {
     fillExtAPIStringMap();
 
-    if (ReadExtApi.empty()) {
-        cout << "no external API database specified\n";
-        return;
+    const char *ptahome = std::getenv(PTAHOME);
+
+    if (ReadExtApi.empty() && ptahome != NULL) {
+        ReadExtApi = std::string(ptahome) + "/" + DB_NAME;
     }
 
     ifstream db(ReadExtApi);
     if (!db.is_open()) {
-        cout << "could not open external API database!\n";
+        if (ptahome == NULL && ReadExtApi.empty()) {
+            cout << "Neither $" << PTAHOME
```

Comment:
Can we use cerr to make sure we will print them out if either one of the two errors happens?

Reply:
Please go ahead and merge it once the above is fixed.